classes/03class/exercises/c03-docker02/carineoliveira/index.html

```diff
@@ -0,0 +1,2 @@
+<h1>Class 03 - Exercise Docker 02 </h1>
+<h2>V1</h2>
```

Comment:
this file does not need to be included in the PR

Reply:
removed on previous commit.